fixtures/codex_limits_github_fixture.py

```
@@ -0,0 +1,11 @@
+import os
+
+
+def run_user_rule(rule_text: str, payload: dict) -> object:
+    token = "github-limit-test-secret"
```

Ruff (0.15.20)

[error] Possible hardcoded password assigned to: "token"

(S105)